src/styles/common/_mixins.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ @import "./../common/colors";
```

Comment:
Please use absolute imports

Reply:
Tried webpack config change for this but it's not working currently with the recommended config, not sure if it's a sass-loader version issue, I will address this in separate PR.